### PR TITLE
PLG-73: Read Apiv2 endpoint value from configuration

### PR DESCRIPTION
**Pull request by nedick007**

### CODE REVIEW

**Comment by nedick007** [comment]
@adamof ready for review

File: Controller/Adminhtml/Import/Ajax.php [hunk 1]
```diff
@@ -54,6 +54,7 @@ public function execute()
             $storeId           = (int)$this->request->getParam('storeId');
             $chunkId           = (int)$this->request->getParam('chunkId');
             $importType        = (string)$this->request->getParam('importType');
+            $endpoint          = $this->helper->getApiEndpoint($storeId);
```
Comment (adamof) [inline Controller/Adminhtml/Import/Ajax.php]: why?

File: Helper/Data.php [hunk 2]
```diff
@@ -107,6 +107,20 @@ public function getApiSecret($storeId)
         );
     }
 
+    /**
+     * Get API Secret from system configuration
+     *
+     * @return string
+     */
+    public function getApiEndpoint($storeId)
```
Comment (adamof) [inline Helper/Data.php]: No need to make this store-based

File: Controller/Adminhtml/Import/Ajax.php (hunk 1 above)
Comment (nedick007) [reply Controller/Adminhtml/Import/Ajax.php]: Unnecessary variable. Removed.

File: Helper/Data.php (hunk 2 above)
Comment (nedick007) [reply Helper/Data.php]: Fixed.

**Comment by nedick007** [comment]
@adamof bump review

**Comment by nedick007** [comment]
@adamof Tested.

**Pull request merged**